event.go

```diff
@@ -26,7 +26,10 @@ import (
 
 // List of built-in account event types.
 const (
-	EventAccountCreated string = "flow.AccountCreated"
+	EventAccountCreated     string = "flow.AccountCreated"
+	EventAccountAdded       string = "flow.AccountKeyAdded"
+	EventAccountKeyRemoved  string = "flow.AccountKeyRemoved"
+	EventAccountCodeUpdated string = "flow.AccountCodeUpdated"
```

Comment:
This event was replaced by three more granular events, see https://github.com/onflow/cadence/blob/v0.13.0/runtime/stdlib/flow.go#L354-L373: 

```suggestion
	EventAccountContractAdded   string = "flow.AccountContractAdded"
	EventAccountContractUpdated string = "flow.AccountContractUpdated"
	EventAccountContractRemoved string = "flow.AccountContractRemoved"
```